woocommerce/woocommerce-ios
# Accessibility: Added VoiceOver support to revenue chart

This PR adds VoiceOver support to the store revenue chart on the My Store tab:

<img width="994" alt="napkin 92 08-30-18 4 30 47 pm" src="https://user-images.githubusercontent.com/154014/44880478-2d3cab00-ac72-11e8-8a17-9812462b527a.png">

There are 3 main elements here:
* The Y-Axis: VoiceOver describes the min and max values
* The X-Axis: VoiceOver describes the min and max values
* The main chart itself: A summary of every non-zero bar in the chart

Regarding the chart summary, I am not sure if it is considered a best practice to skip the 0-value bars, but I did so in the name of brevity. I'm willing to change anything with this summary if desired.

## To Test

### Preconditions:
1. Make sure the app builds and runs and the unit tests are ✅ 
2. On a device, turn on VoiceOver access in your triple-click menu by visiting General > Accessibility > Accessibility Shortcut (select VoiceOver)

### Testing
1. Open the dashboard and wait for data to load in the chart — Verify VoiceOver works on the Y-Axis, X-Axis, and chart itself
2. Rinse and repeat ☝️ for each chart
3. Rotate the device to landscape mode and test steps 1 & 2 again
4. Turn off VoiceOver and verify the individual bars are selectable and display the `ChartMarker` UI like always.

@mindgraffiti would you mind taking a peek? Thanks!
